code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -501,7 +522,8 @@
 	item_state = "stool_bamboo"
 	origin_type = /obj/structure/chair/stool/bamboo
 
-/obj/item/chair/attack_self(mob/user)
+/obj/item/chair/AltClick(mob/user)
+	. = ..()
 	plant(user)
```

Comment:
Did you test to make sure that players can't remotely (i.g. from more than 1 tile away) plant the chair just by alt clicking it from a distance?